internal/nginx-meshctl/commands/remove.go

```diff
@@ -226,14 +239,14 @@ func postHelmCleanup(k8sClient k8s.Client, deleteNamespace bool) (bool, error) {
 
 // removes all custom CRDs.
 func removeCRDs(ctx context.Context, k8sClient k8s.Client) (bool, error) {
-	client := k8sClient.APIExtensionClientSet().ApiextensionsV1().CustomResourceDefinitions()
-	crds, err := client.List(ctx, metav1.ListOptions{LabelSelector: "app.kubernetes.io/part-of=nginx-service-mesh"})
+	cli := k8sClient.APIExtensionClientSet().ApiextensionsV1().CustomResourceDefinitions()
+	crds, err := cli.List(ctx, metav1.ListOptions{LabelSelector: "app.kubernetes.io/part-of=nginx-service-mesh"})
```

Comment:
Nit: `cli` doesn't make it clear what this variable is. Maybe `crdInterface` or `crd` would be better? This is a function scoped variable though and it's not difficult see what it does from context so it's fine either way.